ts/src/static_dependencies/dydx-v4-client/helpers.ts

```diff
@@ -4,12 +4,19 @@
 * and run the transpile command or yarn proto command to regenerate this bundle.
 */
 
-import _m0 from "protobufjs/minimal.js";
+let _m0: any;
+try {
+    // import _m0 from "protobufjs/minimal.js";
+    _m0 = require("protobufjs/minimal.js");
```

Comment:
@carlosmiei IMHO, we may need to change to import, eg `(await import ("protobufjs/minimal.js")).default;`.